### DISCUSSION TITLE
Sujet / Thema 9 –– In welchen Anwendungsfällen bringen OGD Mehrwert?

### DISCUSSION THREAD

**Discussion by schenardi** [GeoUnconference/discussions#24] [Ideen / Idées]
Um das OGD Angebot zu schärfen bin ich der Meinung, dass der Mehrwert gegenüber der Konkurrenz untersucht werden soll. 
* Wo nützen mir welche Swisstopodaten resp. -karten mehr als die aktuell üblichen? 
* Was sind die wirklichen Anwendungsfälle als User bei denen mir diese Daten Mehrnutzen bringen?
* Können daraus ev. Personas abgeleitet werden?
* Können daraus Schlüsse gezogen werden, welche und wie diese Daten bereitgestellt werden sollten?

Im Gegenzug kann auch untersucht werden, wo die OGD Daten schon andere vorhandene Daten ergänzen können und was benötigt wird, um das effizient und zielgerichtet zu tun (z.B. Dienste, Nutzungsvereinbarungen etc).

**Comment by rastrau** [comment]
Aus Nutzendensicht fände ich es interessant, den Mehrwert bzw. Use Cases, in denen dieser (am besten) zum Tragen kommt, herauszuschälen – für Swisstopo und andere amtliche Daten. Wäre das auch aus Anbieter- bzw. Betreibersicht interessant, @davidoesch @edigonzales?

**Comment by edigonzales** [comment]
Ja, Use Cases wären interessant. Gewisse Dinge gehen ja nur mit amtlichen Daten.

**Reply by rastrau** [reply]
Ich habe als Reaktion auf Dein Feedback, @edigonzales, den Titel verallgemeinert (nicht nur Swisstopo-OGD). Ich hoffe, das ist in Ordnung, @schenardi?

**Comment by davidoesch** [comment]
Beispiele von amtlichen Daten in der Anwendung: https://opendata.swiss/de/showcase auch einige mit swisstopo Daten .Challenge : die Uses cases Liste aktuell zu halten.

Welche amtlichen Daten bringen einen signifikanten Mehrwert? Das sind die sogenannten  high Value datasets - im geo Bereich sind sie hier gut erläutert : https://eurogeographics.org/news/high-value-datasets/ auch die Schweiz ist bei den HVD Initiative dabei - und sich auf diese zu konzentrieren macht Sinn

**Comment by sfkeller** [comment]
Ich habe folgende User Stories mit "Use Case"-Charakter:

A1. Als User (oder GIS-Fachmann) möchte ich die (Grundstück-)Gebietseinteilung (Polygone) der Eigentümerkategorien (z.B. Gemeinde, Zivilgemeinde, Korporation, Kanton, Bund, SBB, Post, Swisscom, Private) erhalten.

A2. Als User (oder Service-User) möchte ich an einen kostenlosen und frei zugänglichen Service eine Schweizer Koordinaten schicken und als Antwort die Eigentümerkategorie (z.B. Gemeinde, Zivilgemeinde, Korporation, Kanton, Bund, SBB, Post, Swisscom, Private) erhalten.

B1. Als User (oder Service-User) will ich über einen kostenlosen und frei zugänglichen Service zu einer Schweizer Koordinaten eine Identifikation (E-Mail oder Post-Adresse) der Grundstückeigentümer/Verwaltungen erhalten, damit ich ihm/ihr eine Mitteilung senden kann, z.B. mit einem Vorschlag zum Energiesparen, Spielplatz-Dach bauen, Sitzbank aufstellen, (etc.). 

B2: Als User (oder Service-User) will ich an einen kostenlosen und frei zugänglichen Service eine Mitteilung mit Schweizer Koordinaten schicken, damit der Service diese Mitteilung dann an die Grundstückeigentümer/Verwaltungen weiterleitet (und mir den Empfang bestätigt).

Bemerkung: Ich suche eine pragmatische Lösung innert Monaten, auch wenn sie nicht 100% perfekt ist (und z.B. Private noch nicht einschliesst). Die Daten zu den obigen Use Cases sind wohl im Grundbuch und/oder GWR zu suchen. Die Grundbuchverordnung (GBV 2011) würde das seit 10(!) Jahren zulassen (vgl. Art. 26 + 27). Offenbar gibt es epische Diskussionen nur schon was eine Grundstückeigentümer-Identifikation sein soll (was wenn nicht E-Mail und/oder Post-Adresse?).

**Reply by davidoesch** [reply]
Danke @sfkeller finde die use cases treffen genau die Nutzerbedürfnisse
gibt übrigens ein use case template https://github.com/GeoUnconference/discussions/issues/13  duplizieren und erfassen

GWR ist über die Aapi.geo.admin.ch abrufbar---

**Reply by sfkeller** [reply]
> GWR ist über die Aapi.geoa.dmin.ch abrufbar---

Leider fehlen da IMHO die verlangten Angaben (weder Eigntümer-Identifikator noch Eigentümerkategorie ) zu den oben erwähnten Use Cases A1,A2,B1,B2.

**Reply by davidoesch** [reply]
Die müssten bei den entsprechenden (kantonalen) stellen verfügbar gemacht werden

**Reply by sfkeller** [reply]
> Die müssten bei den entsprechenden (kantonalen) stellen verfügbar gemacht werden

Genau. 
So schreibt es u.a. auch das GBV im Falle der Eigentümer:innen-Identität. 
Die Kantone scheinen sich aber nicht zu "getrauen". Der Konjunktiv ist hier entscheidend :-)

**Comment by schenardi** [comment]
Resultate aus GeoUnconference #2 vom 09.09.2021 
Gruppe: Nicolas Lenz, Sebastian Büttler, @fuerstenberger , Luke Seelenbinder, @schenardi 

# Liste von Anwendungsfällen entstanden in der Diskussion
![IMG_20210909_144912 (1)](https://user-images.githubusercontent.com/22298646/132699212-b1f1b424-7f16-4224-acbd-e5bc42e6fd23.jpg)

Es hat sich gezeigt, dass es sich lohnt, konsequent den Fokus auf dem Bedürfnis des Anwenders zu lassen und stetig darauf bedacht zu sein, in der Diskussion nicht auf technische Lösungsansätze abzuschweifen. Die Zuhilfenahme einer Vorlage #13 ist sehr hilfreich und absolut zu empfehlen, da nur so auch sofort hinterfragt wird, was eigentlich genau benötigt wird und v.a. auch weshalb.

## Historie als wichtiges Datenmerkmal
Eine Erkenntnis bei der Betrachtung der verschiedenen Anwendungsfälle: Zeit resp. historische Daten spielen eine immer wichtigere Rolle. Z.B. bei der Bestimmung von gewachsenem Terrain aus alten Daten oder bei der Vorhersage von Informationen z.B. durch Machine Learning.

**Reply by JonasAeby** [reply]
**Use Case: Gewachsenes Terrain abfragen**

Als Architekt will ich das natürlich gewachsene Terrain auf einem Grundstück kennen, damit ich amtliche Höhenkote für die maximale Gebäudehöhe kenne.

**Comment by rastrau** [comment]
Die Gruppe hat in [einem Google Doc](https://docs.google.com/document/d/1n-oErd53ZQ1YAvzt728ngj0v5j6hBYwvalz2YTuV67A/edit#heading=h.bzlcaoesmtuz) zwei Use Cases ausformuliert bzw. begonnen auszuformulieren: 

- Synthetische Daten generieren/ableiten bei Lücken
- Gewachsenes Terrain abfragen

Dazu wurde folgendes Template benutzt:

As a User: *User Dev, Specialist, Individual, ALL.*

I want (purpose): *Purpose for using the product.*

Because (Trigger): *Reason that causes the user to use the product in the specific way.*

Pre-Conditions: *State of things before the user begins using the product.*

Result: *State of things after the user had completed using the product. Deliverables*

How To Demo: *Step by step for user to complete the task with the product.*

Assumptions: *Assumptions or stipulations made in support of the use case.*

Miscellaneous: *Additional comments and notes about the use case.*

Der Vollständigkeit halbe kopiere ich die beiden Use-Case-Entwürfe als separate Einträge hier in den Thread →

**Comment by rastrau** [comment]
## Use Case: Gewachsenes Terrain abfragen

**Als Architekt will ich das natürlich gewachsene Terrain auf einem Grundstück kennen, damit ich amtliche Höhenkote für die maximale Gebäudehöhe kenne.**

Pre-Conditions:

- *Ich kenne die Parzelle*
- *Auf der Parzelle ist ein Bauvorhaben geplant* 

Result: *DTM der betroffenen Parzelle mit dem natürlich gewachsenen Terrain (wie es vor früheren Bauprojekten war). Die Gemeinde legt das gewachsene Terrain fest.*

How To Demo: 

- *Alte Bilder, alte AV-Pläne oder Karten konsultieren*
- *Befragung von Alt- Eingesessenen*
- Festlegung des gewachsenen Terrains (amtliche Bestätigung)
- *Erstellung eines DTM-Datensatzes*

Assumptions: 

Miscellaneous: *Denkbar ist auch die Erstellung eines grossflächigen Datensatzes inkl. Historisierung.*

**Comment by rastrau** [comment]
## Use Case: Synthetische Daten generieren/ableiten bei Lücken

**Als Versicherungmitarbeiter möchte ich bei einer Abfrage in der Gefahrenkarte auch bei einer Datenlücke eine qualifizierte Aussage erhalten, um eine Risikobeurteilung für ein Grundstück vorzunehmen.**

As a User: 

I want (purpose): 

Because (Trigger): 

Pre-Conditions: 

Result: 

How To Demo:

Assumptions: 

Miscellaneous:
